DEVELOPER_GUIDE.md

```diff
@@ -64,6 +64,22 @@ $ yarn start
 When the server is up and ready, click on the link displayed in your terminal to
 access it.
 
+### Building the archives
+
+To build the archives for each platform,  run the following:
```

Comment:
Is "archives" the correct word here?

Reply:
The last task is called 
``` 
info [  opensearch-dashboards  ] Creating the archives for each platform
```

Shall we change `archives` to `artifacts`? Maybe that sound better?

Reply:
Yeah, I believe `artifacts` is the correct term, right?

Reply:
Yes. Will fix this.